DCS-SimpleTextToSpeech.lua

```diff
@@ -138,16 +171,30 @@ function STTS.TextToSpeech(message,freqs,modulations, volume,name, coalition,poi
         lon = STTS.round(lon,4)
         alt = math.floor(alt)
 
-         cmd = cmd .. string.format(" -L %s -O %s -A %s",lat,lon,alt)        
+        cmd = cmd .. string.format(" -L %s -O %s -A %s",lat,lon,alt)        
     end
 
-    if string.len(cmd) >= 260 then
-        env.warn("[DCS-STTS] TextToSpeech Command is over the 260 character limit and MAY NOT WORK. Reduce the number of parameters or path length")
+    local inlineText = string.format(" -t \"%s\"",message)
+
+    if string.len(cmd) + string.len(inlineText) > 255 then
+        local filename = "tmp_" .. STTS.uuid() .. ".txt"
+        local script = io.open(STTS.DIRECTORY .. "\\" .. filename,"w+")
+        script:write(message)
+        script:close()
+        cmd = cmd .. string.format(" -I \"%s\"",filename)
+        timer.scheduleFunction(os.remove, STTS.DIRECTORY .. "\\" .. filename, timer.getTime() + 5) 
+    else
+        cmd = cmd .. inlineText
     end
 
-    env.info("[DCS-STTS] TextToSpeech Command :\n" .. cmd.."\n")
+    if string.len(cmd) > 255 then
```

Comment:
This bit is here because if you supply position, multiple freqs and say googleTTS data you may exceed the command char limit. this will alert you that is happening (as nothing will happen with the script)